modules/installation/nav-installation-guide.adoc

```diff
@@ -15,14 +20,24 @@ ifeval::[{suma-content} == true]
 *** xref:install-server-unified.adoc[Install Server]
 *** xref:install-proxy-unified.adoc[Install Proxy]
 endif::[]
+ifeval::[{suma-content} == true]
 *** xref:install-vm.adoc[Install Server in a Virtual Machine with JeOS]
+endif::[]
+ifeval::[{uyuni-content} == true]
+*** xref:install-uyuni.adoc[Install Uyuni Server with openSUSE]
+endif::[]
 // better disabling for now; needs a complete overhaul
 // 2019-06-04, ke :
 // *** xref:install-server.adoc[Install Server from Packages]
 //*** xref:install-proxy.adoc[Install Proxy from Packages]
 *** xref:install-ibmz.adoc[Install on IBM Z]
 ** Setting Up
+ifeval::[{suma-content} == true]
```

Comment:
IBM Z above is not supported for Uyuni. Only x86_64.

Reply:
added